lago/plugins/__init__.py

```diff
@@ -58,7 +61,12 @@ def load_plugins(namespace, instantiate=True):
     Returns:
         dict of str, object: Returns the list of loaded plugins
     """
-    mgr = ExtensionManager(namespace=namespace, )
+    mgr = ExtensionManager(namespace=namespace,
+                           on_load_failure_callback=(
+                               lambda manager, ep, err: LOGGER.debug(
```

Comment:
Usually if you are not going to use one of the params, you set it to '_', like:

`lambda _, ep, err:...`

also, try to be consistent with the indenting (if you are breaking the line after the '(' then do so always).

Though as long as it passes the yapf check I'll merge ;)